tests/uefistored/test_secure_boot.py

```diff
@@ -413,12 +417,19 @@ def test_pool_certs_absent_and_disk_certs_present(self, uefi_vm):
         disk_auths = generate_keys(as_dict=True)
         logging.info("Installing certs to hosts disks")
         self.install_certs_to_disks(vm.host.pool, disk_auths, ['PK', 'KEK', 'db', 'dbx'])
-        # start a VM so that certs may be synced to disk if appropriate
+        # Make sure certs are synced to disk
+        if vm.host.xcp_version > '8.2.1':
+            vm.host.restart_toolstack(verify=True)
         vm.start()
```

Comment:
I think we probably don't need to start a VM at all anymore in TestPoolToDiskCertInheritance (except maybe in one dedicated test where we'd test that starting the VM without a toolstack restart does *not* update the certs on disk?)

Reply:
We need the start to get the `resident_on` later in the code.

Reply:
Unless you use the `host` fixture